circle.yml

```diff
@@ -69,7 +69,7 @@ commands:
     - run:
         name: "Run smoketest with fizzy-spectests"
         working_directory: ~/build
-        command: bin/fizzy-spectests --skip-validation ~/project/test/spectests/smoketest
+        command: bin/fizzy-spectests ~/project/test/spectests/smoketest
```

Comment:
Based on the coverage you may want to run it with both options.

Reply:
Perhaps even three times, one more with failures ignored